client.go

```diff
@@ -429,7 +437,6 @@ func (c *Client) stream(method, path string, streamOptions streamOptions) error
 		}
 		return err
 	}
-	defer resp.Body.Close()
```

Comment:
Needed to move Body.Close() because defers are Last-in, First-out and we want the connection closed before closing the body.